### DISCUSSION TITLE
Renaming `todoist` to `todoist-cli`

### DISCUSSION THREAD

**Discussion by kenliu** [sachaos/todoist#217] [General]
Hi todoist users, in https://github.com/sachaos/todoist/issues/183 there is a proposal to rename the binary of this project to something else to prevent a naming conflict with the official Todoist GUI application. 

@sachaos has suggested that this makes sense so I'm opening up for discussion what the new name should be: for sake of argument I'm proposing it be renamed to `todoist-cli` but would like to hear from other folks if there are ideas for a better name.

Let the bikeshedding begin!

**Comment by kenliu** [comment]
I did some research into naming conflicts and I found two other projects named `todoist-cli`, one is archived already (https://pkg.go.dev/github.com/cguy/todoist-cli) and the other is a python package (https://github.com/csu/todoist-cli) that seems mostly inactive and is run by executing `todoist.py`. So overall there shouldn't be any other major conflicts with renaming to `todoist-cli`.

**Comment by HacDan** [comment]
While I think `todoist-cli` would be a great way to avoid naming conflicts, I feel separating from the Todoist brand may be a better choice to avoid affiliation with the Doist brand. 

I don't have any suggestions to add, but I feel this point should be made.

**Comment by psethwick** [comment]
fwiw, the AUR package already uses `todoist-cli` for this package: [todoist on aur](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=todoist)
